lldb/test/Shell/helper/build.py

```diff
@@ -519,9 +519,9 @@ def _find_windows_sdk_in_registry_view(self, view):
 
             # Windows SDK version numbers consist of 4 dotted components, so we
             # have to use LooseVersion, as StrictVersion supports 3 or fewer.
```

Comment:
Does this comment still apply? Are LooseVersion and StrictVersion still a thing we use here?

Reply:
No, it doesn't. StrictVersion was still being used elsewhere in the file, which I've replaced as well.